lsusb.c

```diff
@@ -105,7 +105,6 @@
 
 #define VERBLEVEL_DEFAULT 0	/* 0 gives lspci behaviour; 1, lsusb-0.9 */
 
-#define CTRL_RETRIES	 2
```

Comment:
Why delete this line?

Reply:
I know it's not currently used, but it's not part of this change...

Reply:
It was not even used in initial commit...

Reply:
Greg, sorry - I should have split "remove RETRIES constant" out to a separate patch and have done so in my branch now. PTAL?
(And geezuz...and I still can't get SSH pushes to work with github... what a PITA.